manifests/kots-app.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kots.io/v1beta1
+kind: Application
+metadata:
+  name: Carto
+spec:
+  title: CARTO
+  icon: https://upload.wikimedia.org/wikipedia/commons/thumb/1/13/CARTO-logo.svg/1280px-CARTO-logo.svg.png
+  allowRollback: false
```

Comment:
Here is the line ! ! 
I know that we had the option to enable/disable rollback but I didn't remember it